include/eal.h

```diff
@@ -0,0 +1,21 @@
+#ifndef EAL_H
+#define EAL_H
+
+#include <vector>
+#include <string.h>
+
+using namespace std;
+
+vector<double> getPosition(vector<vector<double>> coords, vector<double> coeffs, char *interp, double time);
+vector<double> getVelocity(vector<vector<double>> coords, vector<double> coeffs, char *interp, double time, bool interpolation);
```

Comment:
Why are both coordinates and coefficients being passed in? I'd expect coordinates and times to be passed in. Also, the input data vectors should be passed as constant references to avoid excessive copying.